src/js/display/displayPost.js

```diff
@@ -0,0 +1,77 @@
+import { fetchGetPostById } from "../request/fetchGetPostById.js";
+import { formatDateAndTime } from "/src/js/components/formatDateAndTime.js";
+import { formatTags } from "/src/js/components/formatTag.js";
+
+
+export async function displayPost() {
+    try {
+        // Extract the post ID from the URL
```

Comment:
I see that you are repeating in other places. Don’t you think having it as a util function would be better?